airbyte-cdk/java/airbyte-cdk/dependencies/build.gradle

```diff
@@ -238,10 +235,18 @@ tasks.named('compileJava').configure {
 tasks.named('compileTestJava').configure {
     dependsOn generate
 }
+<<<<<<< HEAD
 
 tasks.named('compileKotlin').configure {
     dependsOn generate
 }
 tasks.named('compileTestJava').configure {
     dependsOn generate
 }
+
+// uses afterEvaluate because at configuration time, the kspKotlin task does not exist.
+afterEvaluate {
+    tasks.named('kspKotlin').configure {
+        mustRunAfter generateApiServer, generateApiClient, generateApiDocs
+    }
+}
```

Comment:
This needs to be a global rule. IIRC the repo already relies on `generate` being the catch-all code-generation task. I think I already wrote about this in more detail in another review of another PR.

Reply:
This should go away.  There is no need for us to generate the Airbyte API client here.  Instead, we should rely on the version published by the platform so that we stay in sync with it.

Reply:
Good call. It wasn't that hard: https://github.com/airbytehq/airbyte/pull/36610
Not sure why I didn't do it the first time around.

Reply:
I'll mark this as resolved for the time being, on the assumption that my PR will land before yours, so as to not block you.